.github/workflows/pypi-release.yml

```diff
@@ -2,157 +2,211 @@
 # After releasing to GitHub, release to PyPI
 #
 # Inputs:
-#   tag: tag of release in the format v1.2.3rc1
-#   testing: true is releasing to test PyPI, otherwise false
-# 
+#   version_number: The release version number (i.e. 1.0.0b1, 1.2.3rc2, 1.0.0)
+#   test_run      : Test run (True - release to Test PyPI, False - release to PyPI)
```

Comment:
```suggestion
#   test_run      : Test run (true - release to Test PyPI, false - release to PyPI)
```

Reply:
Fixed in 97eaee50b0f8f667fc03b1bbbb76288fb0cee9ad